Player.py

```diff
@@ -0,0 +1,32 @@
+class Player:
+    def __init__(self) -> None:
+        self.health = 100
+        self.energy = 100
+        self.inventory = []
+        self.skills = []
+        self.coins = 0
+        pass
+    
+    def getItem(self, item) -> None:
```

Comment:
Change this method to 'addItem' or 'acquireItem', so that it's clear that it's supposed to add an item to a list, rather than get item information from a list.

Reply:
ok done